src/App.jsx

```diff
@@ -207,20 +215,42 @@ function App() {
           }}
           style={{
             cursor: "pointer",
+            marginLeft: "4%"
           }}
         />
 
         {user ? (
           <>
+
+            <Button onClick={() => setOpenNewUpload(true)} sx={buttonStyle}>
```

Comment:
why's there duplicate code/button? I think while resolving conflicts you mistakenly added both parts of conflicted code